sys_util/src/struct_util.rs

```diff
@@ -6,13 +6,23 @@
 // found in the THIRD-PARTY file.
 
 use std;
-use std::io::Read;
+use std::fmt;
+use std::io::{Error as IoError, ErrorKind, Read};
```

Comment:
Let's use io::Error instead of exporting it as IoError. Some IDEs are having difficulties in tracing where `IoError` is defined.

`ErrorKind` is not needed.

Reply:
@andreeaflorescu , for `Error` and `Read` is both used, would you mind to import `io` on head of file:

```rust
use std::io
```

And then call them as `io::Error` and `io::Read` in this file? They look more graceful.